storage/innobase/include/os0file.h

```diff
@@ -369,6 +369,12 @@ static const ulint ENCRYPTION_INFO_SIZE_V2 = (ENCRYPTION_MAGIC_SIZE \
 					 + ENCRYPTION_SERVER_UUID_LEN \
 					 + 2 * sizeof(ulint));
 
+/** Default master key for bootstrap */
+static const char ENCRYPTION_DEFAULT_MASTER_KEY[] = "DefaultMasterKey";
+
+/** Default master key id for bootstrap */
+static const ulint ENCRYPTION_DEFAULT_MASTER_KEY_ID = 0;
```

Comment:
This one is used in a single source file. Move there? Easy to change later as needed.